server/matchmaker/matchmaker_queue.py

```diff
@@ -58,51 +116,9 @@ def to_dict(self):
         """
         return {
             'queue_name': self.queue_name,
-            'boundary_80s': [search.boundary_80 for player, search in self.queue.items()],
-            'boundary_75s': [search.boundary_75 for player, search in self.queue.items()]
+            'boundary_80s': [search.boundary_80 for search in self.queue.values()],
+            'boundary_75s': [search.boundary_75 for search in self.queue.values()]
```

Comment:
The values are different. Notice the first is `search.boundary_80` and the second is `search.boundary_75`. AFAIK these are to let clients know when other people of similar skill are searching ladder. That's how the Java client sends those annoying 1v1 notifications, and how the py client makes the little banner appear for quick joining a game.

Not sure why there are to different boundaries though. Maybe the 75 one is for new players since they will have lower quality matches.